testing/tests/ui/no-such-escaper.stderr

```diff
@@ -0,0 +1,17 @@
+error: invalid escaper 'latex' for `escape` filter
+ --> LocalEscaper.html:1:38
+       "text|escape(\"latex\")}}`."
+ --> tests/ui/no-such-escaper.rs:3:10
+  |
+3 | #[derive(Template)]
+  |          ^^^^^^^^
+  |
+  = note: this error originates in the derive macro `Template` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: no escaper defined for extension 'tex'
```

Comment:
Thinking more about this error message, I don't think it makes much sense, especially for users who just did a typo. What do you think about `unknown extension 'tex'. The available extensions are 'x', 'y', 'z'`. What do you think?

Reply:
Good idea! Added.